### PR TITLE
Clarification of supported versions

### PR DESCRIPTION
**Pull request by YusukeHosonuma**
Specify support platform versions to `Package.swift` and `.podspec`.
And update README.

All versions are obeyed to [os_log](https://developer.apple.com/documentation/os/os_log).

### CODE REVIEW

File: Sources/Public/CombineExtension.swift [hunk 1]
```diff
@@ -50,6 +50,9 @@
             format: CombineOperatorOption.Format = .multiline,
             to: Output
         ) -> Publishers.HandleEvents<Self> {
+            var option = Pretty.sharedOption
```
Comment (YusukeHosonuma) [inline Sources/Public/CombineExtension.swift:53]: Capture variables are should declaration before closure in Swift 5.1.

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -1,5 +1,13 @@
 name: Test
 
+#
+# Test strategy
+# 
+# - unit-test:        Run all tests on latest macOS and Xcode.
+# - unit-test-linux:  Run tests on Linux (Swift 5.1) - Not include the Combine extension tests.
+# - integration-test: Run integration test by Xcode 11.3.1 and Swift 5.1.3 with CocoaPods.
```
Comment (YusukeHosonuma) [inline .github/workflows/main.yml:8]: This strategy is not covered run all unit tests on Xcode 11.3.1 and Swift 5.1.3.
This is a trade-off currently.

File: README.md [hunk 3]
```diff
@@ -465,11 +466,21 @@ Debug.prettyPrint(label: "array", array)
...
+  - watchOS 3.0+
+  - tvOS 10.0+
+
 ## Development
 
 Require:
 
-- Xcode 11.3
+- Xcode 11.3.1
+  - Note: But run tests are failed on macOS 11.0.1, please use `make test` or latest version of Xcode to run unit tests.
```
Comment (YusukeHosonuma) [inline README.md:483]: At least in my development environment.

File: Tests/SwiftPrettyPrintTests/Public/CombineExtensionTests.swift [hunk 4]
```diff
@@ -36,6 +36,7 @@ struct TestError: Error {
     let message =  "This is the error"
 }
 
+@available(macOS 10.15, iOS 13.0, watchOS 6, tvOS 13, *)
```
Comment (sahara-ooga) [inline Tests/SwiftPrettyPrintTests/Public/CombineExtensionTests.swift:39]: 👍🏻

**Review by sahara-ooga — APPROVED** [review]
Thank you for improving what can be quite a pain in the ass!

**Pull request merged**